testing/run_fuchsia_tests.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import argparse
+import glob
+import os
+import re
+import subprocess
+import sys
+
+buildroot_dir = os.path.abspath(os.path.join(os.path.realpath(__file__), '..', '..', '..'))
+out_dir = os.path.join(buildroot_dir, 'out')
+
+def RunCmd(cmd, **kwargs):
+  print(subprocess.check_output(cmd, **kwargs))
+
+def PackageFlutterRunnerTests(fuchsia_tools):
+  package_cmd = [
+      os.path.join(buildroot_dir, 'flutter', 'tools', 'fuchsia', 'gen_package.py'),
+      '--pm-bin',
+      fuchsia_tools.join('pm'),
+      '--package-dir',
+      os.path.join(out_dir, 'fuchsia_debug_x64', 'flutter_runner_tests_far'),
+      '--signing-key',
+      os.path.join(buildroot_dir, 'flutter', 'tools', 'fuchsia', 'development.key'),
+      '--far-name',
+      'flutter_runner_tests',
+  ]
+  RunCmd(package_cmd)
+
+def IsolateFuchsiaTestArtifacts(fuchsia_tools, image_name):
+  """
+  Gets the system image for the current Fuchsia SDK from cloud storage, adds it
+  to an isolated along with the `pm` and `dev_finder` utilities, as well as the
+  flutter_runner_tests and the flutter_runner FAR, and a bash script (in
+  engine.resources/fuchsia-tests.sh) to drive the flutter_ctl.
+  """
+  with MakeTempDir(api, 'isolated') as isolated_dir:
```

Comment:
What is `api` in this context?

Reply:
This file was accidentally put in this patch. I need to remove it (it was an attempt at moving the fuchsia-related code from the LUCI recipe)